milestones/serializers.py

```diff
@@ -0,0 +1,67 @@
+"""
+Data layer serialization operations.  Converts querysets to simple
+python containers (mainly arrays and dicts).
+"""
+from . import models
+
+
+def serialize_milestone(milestone):
+    """
+    Milestone object-to-dict serialization
+    """
+    return {
+        'id': milestone.id,
+        'name': milestone.name,
+        'namespace': milestone.namespace,
+        'description': milestone.description
+    }
+
+
+def serialize_milestone_with_course(course_milestone):
+    """
+    CourseMilestone serialization (composite object)
+    """
+    return {
+        'id': course_milestone.milestone.id,
+        'name': course_milestone.milestone.name,
+        'namespace': course_milestone.milestone.namespace,
+        'description': course_milestone.milestone.description,
+        'course_id': course_milestone.course_id
+    }
+
+
+def serialize_milestone_with_course_content(course_content_milestone):
+    """
+    CourseContentMilestone serialization (composite object)
+    """
+    return {
+        'id': course_content_milestone.milestone.id,
+        'name': course_content_milestone.milestone.name,
+        'namespace': course_content_milestone.milestone.namespace,
+        'description': course_content_milestone.milestone.description,
+        'course_id': course_content_milestone.course_id,
+        'content_id': course_content_milestone.content_id
+    }
+
+
+def serialize_milestones(milestones):
```

Comment:
I think we could definitely toss out the custom serializers in favor of DRF.  I'll need a hand getting it started and it will have to be added in a follow-on round of development.